### PR TITLE
Missing volume scaling in momentum balance

### PR DESCRIPTION
**Pull request by keileg**
## Proposed changes
This PR fixes a missing volume scaling in the gravity force term of the momentum balance equation.

The PR introduces a test that lithostatic stress is reproduced for pure mechanics, and that the calculated displacements have a reasonable magnitude (this can be converted into a convergence test, but this does not seem worth the effort).

The file `test_fluid_mass_balance.py`  is renamed from `test_single_phase_flow.py` with no additional changes.

## Types of changes

What types of changes does this PR introduce to PorePy?
_Put an `x` in the boxes that apply._

- [ ] Minor change (e.g., dependency bumps, broken links).
- [x] Bugfix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [x] Testing (contribution related to testing of existing or new functionality).
- [ ] Documentation (contribution related to adding, improving, or fixing documentation).
- [ ] Maintenance (e.g., improve logic and performance, remove obsolete code).
- [ ] Other:

## Checklist

_Put an `x` in the boxes that apply or explain briefly why the box is not relevant._

- [x] The documentation is up-to-date.
- [x] Static typing is included in the update.
- [x] This PR does not duplicate existing functionality.
- [x] The update is covered by the test suite (including tests added in the PR).
- [ ] If new skipped tests have been introduced in this PR, `pytest` was run with the `--run-skipped` flag.

### CODE REVIEW

**Comment by IvarStefansson** [comment]
I suggest you have a look at the failing tests before I review.

**Comment by keileg** [comment]
Note to self: This PR must be checked with the `pytest all tests ` workflow.

File: src/porepy/models/constitutive_laws.py [hunk 1]
```diff
@@ -3094,7 +3094,7 @@ def gravity_force(
             material: Name of the material. Could be either "fluid" or "solid".
 
         Returns:
-            Cell-wise nd-vector representing the gravity force.
+            Cell-wise nd-vector representing the gravity force [kg*s^-2*m^-(nd-1)].
```
Comment (IvarStefansson) [inline src/porepy/models/constitutive_laws.py:3097]: Can we talk about densities for nd<3, please?

**Review by IvarStefansson — APPROVED** [review]

File: src/porepy/models/constitutive_laws.py (hunk 1 above)
Comment (keileg) [reply src/porepy/models/constitutive_laws.py:3097]: Decision: This should be m^-2, not m^-(nd-1). I will update tutorials/conventions.ipynb.

File: tutorials/conventions.ipynb [hunk 2]
```diff
@@ -80,7 +80,10 @@
     "\n",
     "$\\sigma(\\epsilon(u)) = C : \\epsilon(u) = C : \\frac{1}{2} (\\nabla u + (\\nabla u)^T)$,\n",
     "\n",
-    "where $C$ denotes the stiffness tensor."
+    "where $C$ denotes the stiffness tensor.\n",
+    "\n",
+    "### Length units\n",
```
Comment (IvarStefansson) [inline tutorials/conventions.ipynb:85]:
```suggestion
    "### Spatial dimensions\n",
    PorePy allows simulations in 3, 2 and, for some cases, 1 dimensions. We denote the highest dimension by `nd`. If `nd`$<3$, the units of the equation change accordingly - e.g. kg/s/m for a `nd`=2 mass balance. This implies that the units of parameters entering the equation remain unchanged. For instance, the mass density is still kg/m$^3$, not kg/m$^2$.
```

**Pull request merged**